src/components/MAPI/apps/AppList/utils.tsx

```diff
@@ -10,14 +10,7 @@ import AppsList from 'UI/Display/Apps/AppList/AppsListPage';
 import CatalogLabel from 'UI/Display/Apps/AppList/CatalogLabel';
 import { IFacetOption } from 'UI/Inputs/Facets';
 
-function isAppCatalogVisibleToUsers(
```

Comment:
These are now used outside the `AppList` components, so moving it to a different file